### PR TITLE
Update spring reference yml after release

### PR DESCRIPTION
**Pull request by chenrujun**
As title.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @yiliuTo , @backwind1233 . 
Please help to review this PR when you have time.

**Comment by backwind1233** [comment]
image
@chenrujun do we need change this?

**Comment by chenrujun** [comment]
@backwind1233 , updated.

File: sdk/spring/spring-reference.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@
               }
           springProperties:
             starter: false
-            compatibilityRange: "[2.5.5,2.6.2)"
```
Comment (backwind1233) [inline sdk/spring/spring-reference.yml:100]: I am not sure about the format here, you can ask @yiliuTo to confirm.

Comment (yiliuTo) [reply sdk/spring/spring-reference.yml:100]: It could be "]" as long as we support 2.6.2

**Review by yiliuTo — APPROVED** [review]

**Pull request merged**